src/main/java/seedu/awe/logic/commands/CreateGroupCommand.java

```diff
@@ -47,4 +94,18 @@ public CommandResult execute(Model model) throws CommandException, DuplicateGrou
         model.addGroup(group);
         return new CommandResult(MESSAGE_SUCCESS);
     }
+
+    @Override
+    public boolean equals(Object other) {
+        if (other instanceof CreateGroupCommand) {
```

Comment:
Can this be structured with guard clause instead?

Reply:
Really sharp of you to notice this. Thank you.